api/server/router/local/container.go

```diff
@@ -30,11 +30,12 @@ func (s *router) getContainersJSON(ctx context.Context, w http.ResponseWriter, r
 	}
 
 	config := &daemon.ContainersConfig{
-		All:     httputils.BoolValue(r, "all"),
-		Size:    httputils.BoolValue(r, "size"),
-		Since:   r.Form.Get("since"),
-		Before:  r.Form.Get("before"),
-		Filters: r.Form.Get("filters"),
+		All:         httputils.BoolValue(r, "all"),
+		Size:        httputils.BoolValue(r, "size"),
+		Since:       r.Form.Get("since"),
+		Before:      r.Form.Get("before"),
+		Filters:     r.Form.Get("filters"),
+		IgnoreLinks: httputils.BoolValue(r, "ignoreLinks"),
```

Comment:
I dunno, `trunc` sounds pretty fit for this and doesn't add anything to API. And it means same things from cli point of view. I'd better to try to keep it same if possible and now I don't see any requests for separate flag.